UnitTests/tEqualityResolution.cpp

```diff
@@ -55,7 +55,7 @@ TEST_GENERATION(test_02,
 TEST_GENERATION(test_03,
     Generation::AsymmetricTest()
       .input(     clause({  selected(x != f(a)), selected(p(x))  }))
-      .expected( exactly( clause({  p(f(a))                              })))
+      .expected( exactly())
```

Comment:
Huh?

Reply:
Marton's fix. I think it says that the inference will not trigger (because it does not pass the `lma` check at the end and `lma` is newly on by default.)